<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cart</class>
 <widget class="QWidget" name="Cart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>656</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Franklin Gothic Book</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_cartIsEmpry">
     <property name="text">
      <string>Корзина товаров пуста</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_clearProduct">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить выбранный товар</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_placeOrder">
     <property name="font">
      <font>
       <family>Franklin Gothic Book</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Оформить заказ</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
